src/portal/lib/src/service/CVEWhitelist.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { map, catchError } from "rxjs/operators";
+import { Observable, throwError as observableThrowError } from "rxjs";
+import {SystemCVEWhitelist} from './interface';
+import {HTTP_GET_OPTIONS, HTTP_JSON_OPTIONS} from "../utils";
+
+
+
+
+@Injectable()
+export class CVEWhitelistService {
+  constructor(private http: HttpClient) {
+  }
+
+  /**
+   *  get system CEVWhitelist
+   */
+  getSystemWhitelist(): Observable<SystemCVEWhitelist> {
```

Comment:
list  ?    return Array?

Reply:
it's called  "whitelist", in fact ,it's object type.